ably/ably_suite_test.go

```diff
@@ -30,7 +29,7 @@ var _ = BeforeSuite(func() {
 
 var _ = BeforeEach(func() {
 	client = ably.NewRestClient(testApp.Params)
-	channel = client.Channel("test")
+	channel = client.RestChannel("test")
```

Comment:
It sounds weird to write RestChannel when we already have specified that we instantiated a RestClient. Shouldn't that just be `client.Channel` anyway?